editor/components/block-list/layout.js

```diff
@@ -237,7 +234,10 @@ class BlockListLayout extends Component {
 
 export default connect(
 	( state ) => ( {
-		selectedBlock: getSelectedBlock( state ),
+		// Reference block selection value directly, since current selectors
+		// assume either multi-selection (getMultiSelectedBlocksStartUid) or
+		// singular-selection (getSelectedBlock) exclusively.
+		selectionStartUID: state.blockSelection.start,
```

Comment:
Shouldn't we just add a new selector?

Reply:
Problem is we'd then have to distinguish between three ways to get access to the UID of the start of a selection:

- `getSelectedBlock( state ).uid`
   - When there's only a single block selected
- `getMultiSelectedBlocksStartUid`
   - When there's a multi-selection
- `getSelectionStart`
   - When there's either of the above